### PR TITLE
Add the config.js.go template.

### PR DESCRIPTION
**Pull request by frankban**
It is used to generate the GUI configuration when the GUI is served by Juju.
Also update and rename index.html.go.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4403/
Test PASSed.

File: jujugui/templates/config.js.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var juju_config = {
```
Comment (hatched) [inline jujugui/templates/config.js.go]: Could you add a list of the possible values provided by juju?

**Comment by hatched** [comment]
:+1: Thanks!

File: jujugui/templates/config.js.go (hunk 1 above)
Comment (frankban) [inline jujugui/templates/config.js.go]: Good suggestion Jeff, but no.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**